pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -663,6 +688,75 @@ func GetShortNameMode(ctx *types.SystemContext) (types.ShortNameMode, error) {
 	return config.shortNameMode, err
 }
 
+// GlobalCredentialHelpers returns the global top-level credential helpers.
+// A slice with only AuthenticationFileHelper is returned if no helpers are
+// configured.
```

Comment:
This should not be an API promise. Document what the caller is allowed to do with the return value, not how it is formed.